### PR TITLE
Fixed messages displayed with captains

### LINKED ISSUE #4860: Combat screen, Captain's Options window, wrong text in the info bar at the bottom of the screen
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.
- [X] I've checked that this issue is reproducible on the [**latest snapshot build**](https://ihhub.github.io/fheroes2/INSTALL.html#snapshots-latest-builds).

### Platform

Windows

### Describe the bug

In fheroes2, there is wrong text in the info bar at the bottom of the screen.

In original game:

Captain info bar 001

Captain info bar 003

In fheroes2:

Captain info bar 002

Captain info bar 004

### Save file

Not needed.

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by felix642**
closes #4860

I still need to update the translations.

### CODE REVIEW

File: src/fheroes2/battle/battle_interface.cpp [hunk 1]
```diff
@@ -1066,11 +1074,11 @@ void Battle::Interface::UpdateContourColor()
     ++_contourCycle;
 
     if ( _brightLandType ) {
-        static const uint8_t contourColorTable[] = {108, 115, 122, 129, 122, 115};
+        static const uint8_t contourColorTable[] = { 108, 115, 122, 129, 122, 115 };
```
[the remark this answers is not in the document]
Comment (felix642) [reply src/fheroes2/battle/battle_interface.cpp:1077]: That line was changed because of clang-format. Do you still want me to fix this @ihhub  ?

**Comment by felix642** [comment]
Hi @undef21 and @Arthusppp , can you please help with Russian and Polish translations here? I have added 2 new strings and I would appreciate if you could help me translate them.

File: files/lang/pl.po [hunk 2]
```diff
@@ -482,6 +482,12 @@ msgstr ""
 msgid "View Hero's options"
 msgstr "Opcje bohatera"
 
+msgid "View Captain's options"
+msgstr ""
```
Comment (Arthusppp) [inline files/lang/pl.po:486]: "Opcje kapitana"

File: files/lang/pl.po [hunk 3]
```diff
@@ -482,6 +482,12 @@ msgstr ""
...
+
+msgid "View opposing Captain"
+msgstr ""
```
Comment (Arthusppp) [inline files/lang/pl.po:489]: "Obejrzyj wrogiego kapitana"

File: src/fheroes2/battle/battle_interface.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (yvesgurcan) [reply src/fheroes2/battle/battle_interface.cpp:1077]: I've seen these messages in other PRs. It does not seem like acting on it is necessary. 🙂

**Review by yvesgurcan — APPROVED** [review]
Excellent! What does the original game do for captains in this case?

**Comment by felix642** [comment]
> Excellent! What does the original game do for captains in this case?

Hi @yvesgurcan, you can take a look at the linked issue #4860. @LeHerosInconnu has provided an example with some screenshots from the original game.

File: files/lang/pl.po (hunk 2 above)
Comment (felix642) [reply files/lang/pl.po]: Thank you for your help @Arthusppp :)

**Comment by ihhub** [comment]
Hi @felix642 , can we please make this pull request ready for review? We can skip translations for now.

**Comment by felix642** [comment]
Hi @ihhub, I have updated my PR.

**Comment by felix642** [comment]
There are still a few issues. I will fix the pipeline and notify you when the PR will be ready for reviews.

**Comment by felix642** [comment]
@ihhub I think this PR is ready for reviews.

**Comment by ihhub** [comment]
@felix642 , a bunch of thanks for this change!

**Pull request merged**